### PR TITLE
Upmerge 25-03-2024

### PR DESCRIPTION
**Pull request by gmarull**
test-sdk-nrf: sdk-nrf-pr-14626

### CODE REVIEW

**Comment by tony-le-24** [comment]
We found an issue with the last upmerge-01.03.24 that failed the application FOTA update confirmation with error 4 on our nrf-cloud_rest_fota sample (and other nrf_cloud samples as well). I suspect this commit caused the issue: [https://github.com/nrfconnect/sdk-zephyr/commit/24192740730cd3021114a2e32283443ec18a8ab7](https://github.com/nrfconnect/sdk-zephyr/commit/24192740730cd3021114a2e32283443ec18a8ab7). Please take a look. Thanks

File: samples/crypto/sha256/sample.yaml [hunk 1]
```diff
@@ -5,39 +5,39 @@ tests:
...
   sample.newlib_libc.sha256:
     build_only: true
     extra_args: CONFIG_NEWLIB_LIBC=y
-    platform_allow: nrf5340dk_nrf5340_cpuapp_ns nrf52840dk_nrf52840 nrf54h20dk_nrf54h20_cpuapp
-    integration_platforms:
-      - nrf5340dk_nrf5340_cpuapp_ns
-      - nrf52840dk_nrf52840
-      - nrf54h20dk_nrf54h20_cpuapp
+    platform_allow: nrf5340dk/nrf5340/cpuapp/ns nrf52840dk/nrf52840 nrf54h20dk/nrf54h20/cpuapp
+    integration/platforms:
```
Comment (katgiadla) [inline samples/crypto/sha256/sample.yaml:40]:
```suggestion
    integration_platforms:
```

Please correct this due to Twister issue

**Review by katgiadla — CHANGES_REQUESTED** [review]

File: samples/crypto/sha256/sample.yaml (hunk 1 above)
Comment (gmarull) [reply samples/crypto/sha256/sample.yaml]: done

**Comment by gchwier** [comment]
> We found an issue with the last upmerge-01.03.24 that failed the application FOTA update confirmation with error 4 on our nrf-cloud_rest_fota sample (and other nrf_cloud samples as well). Looked like the `boot_write_img_confirmed` function returned 4. I suspect this commit caused the issue: [nrfconnect/sdk-zephyr@2419274](https://github.com/nrfconnect/sdk-zephyr/commit/24192740730cd3021114a2e32283443ec18a8ab7). Please take a look. Thanks. GitHub issue: [zephyrproject-rtos/zephyr#70764](https://github.com/zephyrproject-rtos/zephyr/issues/70764) JIRA ticket: https://nordicsemi.atlassian.net/browse/NCSDK-26766

please check that PR: https://github.com/nrfconnect/sdk-nrf/pull/14633 (jira https://nordicsemi.atlassian.net/browse/NCSDK-26693 )
After last upmerge there is problem, that `boot_write_img_confirmed()` does not confirm image when PM is used

File: tests/subsys/bluetooth/fast_pair/storage/factory_reset/testcase.yaml [hunk 2]
```diff
@@ -14,43 +14,43 @@ tests:
   fast_pair.storage.factory_reset.no_reboot:
```
Comment (alstrzebonski) [inline tests/subsys/bluetooth/fast_pair/storage/factory_reset/testcase.yaml:14]: Lines 1-14 are not aligned

**Review by alstrzebonski — CHANGES_REQUESTED** [review]
Also tests/subsys/bluetooth/fast_pair/storage/account_key_storage/testcase.yaml is not aligned.

**Comment by hubertmis** [comment]
I'm not sure what happened. I removed myself from the reviewers list and it looks GitHub automatically removed many more reviewers

File: tests/subsys/bluetooth/fast_pair/crypto/testcase.yaml [hunk 3]
```diff
@@ -1,16 +1,21 @@
...
-      - nrf52dk_nrf52832
-      - nrf52840dk_nrf52840
-      - nrf5340dk_nrf5340_cpuapp
+      - nrf52dk/nrf52832
+      - nrf52840dk/nrf52840
+      - nrf5340dk/nrf5340/cpuapp
     extra_args: CONF_FILE=prj_mbedtls.conf
   fast_pair.crypto.oberon:
     platform_allow:
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (gmarull) [reply tests/subsys/bluetooth/fast_pair/crypto/testcase.yaml]: fixed

**Comment by miha-nordic** [comment]
@gmarull  can we add force downstream to check jobs status even without twister build working? Maybe we can fix something in advance.

**Review by alstrzebonski — APPROVED** [review]
ACK on `fast_pair` part

**Comment by SeppoTakalo** [comment]
How does the building on the new Hardware Model V2 works?
```
$ west build -b nrf9160dk/nrf9160/ns
...
CMake Error at /home/seta/src/ncs/zephyr/cmake/modules/boards.cmake:311 (message):
  Board qualifiers `` for board `nrf9160dk` not found.  Please specify a
  valid board target.

  Valid board targets for are:

  nrf9160dk/nrf9160

  nrf9160dk/nrf9160/ns

  nrf9160dk/nrf52840
```

What board name should I use now?

**Review by Balaklaka — APPROVED** [review]

**Comment by MirkoCovizzi** [comment]
There appears to be a failure with regards to linking of modem library due to the updated HWMv2 naming. Looking into it.

**Comment by MirkoCovizzi** [comment]
@gmarull in `nrfxlib` please include the fix for modem library linking, at https://github.com/nrfconnect/sdk-nrfxlib/pull/1278

Thanks

**Comment by jumi-nordic** [comment]
In mcuboot we observed build system issue:
cmake command: 
> cmake -GNinja -DBOARD='nrf52840dk/nrf52840' -Dbuild_folder_path=build_ninja -DCONFIG_BOOTLOADER_MCUBOOT=y -Dmcuboot_CONFIG_MCUBOOT_SERIAL=y -Dmcuboot_CONFIG_MCUBOOT_UART_CONSOLE=n -Dmcuboot_CONFIG_FLASH=y -S /home/jumi/test-fw-nrfconnect-apps/test_objects/ncs/zephyr/samples/hello_world/ -B build_ninja

or west with sysbuild conf:
> west build -b nrf52840dk/nrf52840 -p --sysbuild -- -DCONFIG_BOOTLOADER_MCUBOOT=y -Dmcuboot_CONFIG_MCUBOOT_SERIAL=y -Dmcuboot_CONFIG_MCUBOOT_UART_CONSOLE=n -Dmcuboot_CONFIG_FLASH=y -DSB_CONFIG_BOOTLOADER_MCUBOOT=y

It seems like it doesn't recognize these new qualifiers, cmake error:
```
CMake Error at ncs/zephyr/cmake/modules/boards.cmake:311 (message):
  Board qualifiers '' for board 'nrf52840dk' not found.  Please specify a
  valid board target.

  Valid board targets for are:

  nrf52840dk/nrf52840
```

**Comment by nordicjm** [comment]
> In mcuboot we observed build system issue: cmake command:
> 
> > cmake -GNinja -DBOARD='nrf52840dk/nrf52840' -Dbuild_folder_path=build_ninja -DCONFIG_BOOTLOADER_MCUBOOT=y -Dmcuboot_CONFIG_MCUBOOT_SERIAL=y -Dmcuboot_CONFIG_MCUBOOT_UART_CONSOLE=n -Dmcuboot_CONFIG_FLASH=y -S /home/jumi/test-fw-nrfconnect-apps/test_objects/ncs/zephyr/samples/hello_world/ -B build_ninja
> 
> or west with sysbuild conf:
> 
> > west build -b nrf52840dk/nrf52840 -p --sysbuild -- -DCONFIG_BOOTLOADER_MCUBOOT=y -Dmcuboot_CONFIG_MCUBOOT_SERIAL=y -Dmcuboot_CONFIG_MCUBOOT_UART_CONSOLE=n -Dmcuboot_CONFIG_FLASH=y -DSB_CONFIG_BOOTLOADER_MCUBOOT=y
> 
> It seems like it doesn't recognize these new qualifiers, cmake error:
> 
> ```
> CMake Error at ncs/zephyr/cmake/modules/boards.cmake:311 (message):
>   Board qualifiers '' for board 'nrf52840dk' not found.  Please specify a
>   valid board target.
> 
>   Valid board targets for are:
> 
>   nrf52840dk/nrf52840
> ```

Fix: https://github.com/gmarull/sdk-nrf/pull/4

File: samples/bluetooth/peripheral_lbs/sample.yaml [hunk 4]
```diff
@@ -5,30 +5,31 @@ tests:
...
+      - nrf52dk/nrf52810
+      - nrf52840dk/nrf52811
+      - nrf52833dk/nrf52820
+    platform_allow: nrf52dk/nrf52810 nrf52840dk/nrf52811 nrf52833dk/nrf52820
     tags: bluetooth ci_build
   sample.bluetooth.peripheral_lbs_no_security:
     build_only: true
     extra_args: CONFIG_BT_LBS_SECURITY_ENABLED=n
     integration_platforms:
+<<<<<<< HEAD
```
Comment (gchwier) [inline samples/bluetooth/peripheral_lbs/sample.yaml:32]: merge not completed,
found also in:
./samples/matter/light_bulb/sample.yaml:<<<<<<< HEAD
./samples/matter/light_switch/sample.yaml:<<<<<<< HEAD
./samples/bluetooth/peripheral_hids_mouse/sample.yaml:<<<<<<< HEAD
./samples/bluetooth/peripheral_lbs/sample.yaml:<<<<<<< HEAD

**Comment by gmarull** [comment]
@moksanen please, sign CLA

File: samples/bluetooth/peripheral_lbs/sample.yaml (hunk 4 above)
Comment (gmarull) [reply samples/bluetooth/peripheral_lbs/sample.yaml]: fixed

**Comment by moksanen** [comment]
> @moksanen please, sign CLA

Found it, and done.

**Comment by jtguggedal** [comment]
@gmarull Here's a commit fixing Asset Tracker v2 compilation issue: https://github.com/jtguggedal/fw-nrfconnect-nrf/commit/ad56a3e2a43d006825e0cc0b32af9e0db6736eba

**Comment by jorgenmk** [comment]
@gmarull Here is a commit that fixes building asset tracker for 9160dk@0.7.0: 
https://github.com/nrfconnect/sdk-nrf/pull/14607/commits/44f27fceb02fde15fca4071b78474828b7592c8f

**Comment by trantanen** [comment]
> @gmarull Here's a commit fixing Asset Tracker v2 compilation issue: [jtguggedal@ad56a3e](https://github.com/jtguggedal/fw-nrfconnect-nrf/commit/ad56a3e2a43d006825e0cc0b32af9e0db6736eba)

I guess we'd need similar renaming for the folders under the boards folder of the apps and samples. @gmarull is this something that will be done tree-wide or do we need it for each sample separately?

**Comment by gmarull** [comment]
> > @gmarull Here's a commit fixing Asset Tracker v2 compilation issue: [jtguggedal@ad56a3e](https://github.com/jtguggedal/fw-nrfconnect-nrf/commit/ad56a3e2a43d006825e0cc0b32af9e0db6736eba)
> 
> I guess we'd need similar renaming for the folders under the boards folder of the apps and samples. @gmarull is this something that will be done tree-wide or do we need it for each sample separately?

I see you're using a custom scheme, out of Zephyr standard way of doing things. If that's the case, I'd suggest to script something or tell me what should be changed. Are you sure you want to change to board and not board target when searching for these files? What if a board with cpuapp/cpunet require different sets?

**Comment by jtguggedal** [comment]
> > > @gmarull Here's a commit fixing Asset Tracker v2 compilation issue: [jtguggedal@ad56a3e](https://github.com/jtguggedal/fw-nrfconnect-nrf/commit/ad56a3e2a43d006825e0cc0b32af9e0db6736eba)
> > 
> > 
> > I guess we'd need similar renaming for the folders under the boards folder of the apps and samples. @gmarull is this something that will be done tree-wide or do we need it for each sample separately?
> 
> I see you're using a custom scheme, out of Zephyr standard way of doing things. If that's the case, I'd suggest to script something or tell me what should be changed. Are you sure you want to change to board and not board target when searching for these files? What if a board with cpuapp/cpunet require different sets?

What would be the Zephyr way of doing things in this case? Is there a way I could include the folders based on board target instead? I couldn't find a cmake variable for getting the board target and use that to generate the correct path, so ended up with renaming the folders to the board name instead.

**Comment by gmarull** [comment]
> > > > @gmarull Here's a commit fixing Asset Tracker v2 compilation issue: [jtguggedal@ad56a3e](https://github.com/jtguggedal/fw-nrfconnect-nrf/commit/ad56a3e2a43d006825e0cc0b32af9e0db6736eba)
> > > 
> > > 
> > > I guess we'd need similar renaming for the folders under the boards folder of the apps and samples. @gmarull is this something that will be done tree-wide or do we need it for each sample separately?
> > 
> > 
> > I see you're using a custom scheme, out of Zephyr standard way of doing things. If that's the case, I'd suggest to script something or tell me what should be changed. Are you sure you want to change to board and not board target when searching for these files? What if a board with cpuapp/cpunet require different sets?
> 
> What would be the Zephyr way of doing things in this case? Is there a way I could include the folders based on board target instead? I couldn't find a cmake variable for getting the board target and use that to generate the correct path, so ended up with renaming the folders to the board name instead.

We should likely be using DT for that, not header files. Then standard overlays would just work.

**Comment by jtguggedal** [comment]
> We should likely be using DT for that, not header files. Then standard overlays would just work.

I see. The same pattern as here is used also in other places that use CAF for LED handling, so it should probably be done for all instances. I suggest doing that outside the scope of this PR, though

**Comment by gmarull** [comment]
> > We should likely be using DT for that, not header files. Then standard overlays would just work.
> 
> I see. The same pattern as here is used also in other places that use CAF for LED handling, so it should probably be done for all instances. I suggest doing that outside the scope of this PR, though

Yes, of course, this is a topic not for an upmerge. What I'd suggest is to maybe use `CONFIG_BOARD_TARGET` instead of `CONFIG_BOARD` if the headers have the potential to be different depending on the target core.

**Comment by jtguggedal** [comment]
> > > We should likely be using DT for that, not header files. Then standard overlays would just work.
> > 
> > 
> > I see. The same pattern as here is used also in other places that use CAF for LED handling, so it should probably be done for all instances. I suggest doing that outside the scope of this PR, though
> 
> Yes, of course, this is a topic not for an upmerge. What I'd suggest is to maybe use `CONFIG_BOARD_TARGET` instead of `CONFIG_BOARD` if the headers have the potential to be different depending on the target core.

Yeah, I was looking for a cmake variable that evaluated to for instance `nrf9161dk_nrf9161_ns` when building for `nrf9161dk/nrf9161/ns`, but couldn't find any such variable being generated. `CONFIG_BOARD_TARGET` also includes the revision, so `nrf9161dk@0.9.0/nrf9161` in the example above, so also not directly usable.

Anyway, in the Asset Tracker v2 case, the proposed fix is okay

**Comment by jumi-nordic** [comment]
> In mcuboot we observed build system issue: cmake command:
> 
> > cmake -GNinja -DBOARD='nrf52840dk/nrf52840' -Dbuild_folder_path=build_ninja -DCONFIG_BOOTLOADER_MCUBOOT=y -Dmcuboot_CONFIG_MCUBOOT_SERIAL=y -Dmcuboot_CONFIG_MCUBOOT_UART_CONSOLE=n -Dmcuboot_CONFIG_FLASH=y -S /home/jumi/test-fw-nrfconnect-apps/test_objects/ncs/zephyr/samples/hello_world/ -B build_ninja
> 
> or west with sysbuild conf:
> 
> > west build -b nrf52840dk/nrf52840 -p --sysbuild -- -DCONFIG_BOOTLOADER_MCUBOOT=y -Dmcuboot_CONFIG_MCUBOOT_SERIAL=y -Dmcuboot_CONFIG_MCUBOOT_UART_CONSOLE=n -Dmcuboot_CONFIG_FLASH=y -DSB_CONFIG_BOOTLOADER_MCUBOOT=y
> 
> It seems like it doesn't recognize these new qualifiers, cmake error:
> 
> ```
> CMake Error at ncs/zephyr/cmake/modules/boards.cmake:311 (message):
>   Board qualifiers '' for board 'nrf52840dk' not found.  Please specify a
>   valid board target.
> 
>   Valid board targets for are:
> 
>   nrf52840dk/nrf52840
> ```

The issue is still valid for child  s1_image:
west command: west build -p -b nrf52840dk/nrf52840 -s $NCS/zephyr/samples/hello_world -- -DCONFIG_BUILD_S1_VARIANT=y -DCONFIG_SECURE_BOOT=y -Db0_SB_CRYPTO_CC310_ECDSA_SECP256R1=y -Db0_SB_CRYPTO_CC310_SHA256=y 

`
=== child image s1_image -  begin ===
loading initial cache file build/image_preload.cmake
-- Application: ncs/zephyr/samples/hello_world
-- CMake version: 3.28.4
Loading Zephyr default modules (Zephyr base).
-- Found Python3: venv/bin/python (found suitable version "3.10.12", minimum required is "3.8") found components: Interpreter 
-- Cache files will be written to: .cache/zephyr
-- Zephyr version: 3.6.99 (ncs/zephyr)
-- Found west (found suitable version "1.0.0", minimum required is "0.14.0")
CMake Error at zephyr/cmake/modules/boards.cmake:315 (message):
  Board qualifiers `` for board `nrf52840dk` not found.  Please specify a
  valid board target.

  Valid board targets for are:

  nrf52840dk/nrf52840
`

File: cmake/extensions.cmake [hunk 5]
```diff
@@ -210,7 +210,14 @@ Please provide one of following: CONF_FILES")
     endif()
   endif()
 
-  zephyr_file(CONF_FILES ${NCS_FILE_CONF_FILES}/boards ${NCS_FILE_UNPARSED_ARGUMENTS})
+  set(additional_append)
+
+  if(DEFINED PREPROCESS_ARGS_BOARD)
+    parse_board_components(PREPROCESS_ARGS_BOARD board_name board_revision board_qualifiers)
+    set(additional_append BOARD ${board_name} BOARD_REVISION ${board_revision} BOARD_QUALIFIERS ${board_qualifiers})
+  endif()
```
Comment (tejlmand) [inline cmake/extensions.cmake:218]: To be addressed before merging, see: https://github.com/gmarull/sdk-nrf/pull/13#discussion_r1570646695

File: cmake/multi_image.cmake [hunk 6]
```diff
@@ -335,6 +335,8 @@ function(add_child_image_from_source)
       # Place the result in the CMake cache and remove local scoped variable.
       foreach(file CONF_FILE DTC_OVERLAY_FILE)
         if(DEFINED ${ACI_NAME}_${file})
+          zephyr_file_suffix(${ACI_NAME}_${file} SUFFIX ${${ACI_NAME}_FILE_SUFFIX})
```
Comment (tejlmand) [inline cmake/multi_image.cmake:338]: to be addressed, or an acceptable reason for why to accept this is given.

See also: https://github.com/gmarull/sdk-nrf/pull/13#discussion_r1570631752

**Review by tejlmand — CHANGES_REQUESTED** [review]

**Comment by miha-nordic** [comment]
@gmarull  Can we add "CI-force-downstream" label to enable work on verification site or it conflicts with your plans and make work harder?

**Comment by gmarull** [comment]
> @gmarull Can we add "CI-force-downstream" label to enable work on verification site or it conflicts with your plans and make work harder?

added, note that some noise due to hwmv2 may still appear.

**Comment by jtguggedal** [comment]
It looks like Kconfig options defined in the original `boards/arm/thingy91_nrf9160/Kconfig.board` have been deleted in 1c6d841b9fc04c76ae0f3b7bfca3cf58036884dc. The file should instead just be moved to the new location.
This results in the wrong or no `pm_static*.yml` being used and partitions are misaligned.

File: cmake/multi_image.cmake [hunk 7]
```diff
@@ -317,51 +321,65 @@ function(add_child_image_from_source)
...
+      set(ACI_NAME_CONF_DIR ${config_dir}/child_image/${ACI_NAME})
+      if (NOT ${ACI_NAME}_CONF_FILE)
+        ncs_file(CONF_FILES ${ACI_NAME_CONF_DIR}
+          BOARD ${ACI_BOARD}
+          # Child image always uses the same revision as parent board.
+          BOARD_REVISION ${BOARD_REVISION}
+          KCONF ${ACI_NAME}_CONF_FILE
+          DTS ${ACI_NAME}_DTC_OVERLAY_FILE
+          BUILD ${CONF_FILE_BUILD_TYPE}
+          SUFFIX ${${ACI_NAME}_FILE_SUFFIX}
```
Comment (nordicjm) [inline cmake/multi_image.cmake:337]: Not correct, needs fixing before merge https://github.com/gmarull/sdk-nrf/pull/17#discussion_r1575882251

**Review by nordicjm — CHANGES_REQUESTED** [review]

File: cmake/multi_image.cmake (hunk 7 above)
Comment (gmarull) [reply cmake/multi_image.cmake:337]: @tejlmand

**Review by ktaborowski — CHANGES_REQUESTED** [review]
In sidewalk application ble fails after start on nRF53 (ipc bound timeout)
```
[00:00:00.115,844] <inf> sid_ble: Enable BT
[00:00:01.116,821] <err> bt_hci_driver: Endpoint binding failed with -11
[00:00:01.116,851] <err> bt_hci_core: HCI driver open failed (-11)
[00:00:01.116,851] <err> sid_ble: BT init failed (err -11)
```
It found the board configuration such as:
```
-- Found devicetree overlay: /home/ktab/ncs/v2.6.0/sidewalk/samples/sid_end_device/boards/nrf5340dk_nrf5340_cpuapp.overlay
Merged configuration '/home/ktab/ncs/v2.6.0/sidewalk/samples/sid_end_device/boards/nrf5340dk_nrf5340_cpuapp.conf'
Parsing /home/ktab/ncs/v2.6.0/sidewalk/samples/sid_end_device/child_image/hci_ipc/Kconfig.root
Merged configuration '/home/ktab/ncs/v2.6.0/sidewalk/samples/sid_end_device/child_image/hci_ipc/prj.conf'
```
The issue is not observed on main.
Please have a look https://github.com/nrfconnect/sdk-sidewalk/pull/543
https://jenkins-ncs.nordicsemi.no/job/KRKNG/job/Sidewalk/job/test-sdk-sidewalk/job/sdk-nrf-pr-14626/9/testReport/functional.dfu_bt_tests/nrf53_TestDfuOverBleRegister_5340r1app_tgt_zephyr_fsk_b8ad/Tests___Functional_Tests___Execute_Functional_Tests___test_dfu_bt_basic_on_connected/.

**Review by ktaborowski — CHANGES_REQUESTED** [review]
Can't init ble with Sidewalk after up merge

```
*** Booting nRF Connect SDK v3.5.99-ncs1-7382-gf32566af5714 ***
----------------------------------------------------------------
nrf                  v2.6.99-cs1-85-g2e9e56d27
zephyr               v3.5.99-ncs1-7382-gf32566af57
sidewalk             v2.6.99-cs1-2-gc81b81c
----------------------------------------------------------------
Sidewalk SDK        = 1.16.1.28
build time          = 2024-04-29 08:48:46.068231+00:00
board               = nrf5340dk
----------------------------------------------------------------
[00:00:00.001,556] <inf> application_state: working = true
[00:00:00.071,228] <inf> bt_hci_core: HW Platform: Nordic Semiconductor (0x0002)
[00:00:00.071,289] <inf> bt_hci_core: HW Variant: nRF53x (0x0003)
[00:00:00.071,319] <inf> bt_hci_core: Firmware: Standard Bluetooth controller (0x00) Version 137.63438 Build 2523887658
[00:00:00.108,184] <inf> bt_hci_core: Identity: D2:FE:2D:BD:1C:3F (random)
[00:00:00.108,184] <inf> bt_hci_core: HCI: version 5.4 (0x0d) revision 0x21bc, manufacturer 0x0059
[00:00:00.108,215] <inf> bt_hci_core: LMP: version 5.4 (0x0d) subver 0x21bc
metal: warning:   tx_vq: freeing non-empty virtqueue
metal: warning:   rx_vq: freeing non-empty virtqueue
[00:00:00.113,677] <inf> sidewalk_app: Sidewalk link switch to BLE
[00:00:00.115,814] <inf> sid_ble: Enable BT
[00:00:01.116,790] <err> bt_hci_driver: Endpoint binding failed with -11
[00:00:01.116,821] <err> bt_hci_core: HCI driver open failed (-11)
[00:00:01.116,821] <err> sid_ble: BT init failed (err -11)
```

**Review by edmont — APPROVED** [review]
Zigbee ✅

**Review by shanthanordic — APPROVED** [review]
upmerge scope accepted. Downstream failures are accepted.

File: applications/nrf_desktop/configuration/nrf54l15pdk_nrf54l15_cpuapp/child_image/mcuboot/prj.conf [hunk 8]
```diff
@@ -25,6 +25,7 @@ CONFIG_BOOT_SIGNATURE_KEY_FILE="mcuboot_private.pem"
 
 # Flash
 CONFIG_FLASH=y
+CONFIG_SPI_NOR=n
```
Comment (tejlmand) [inline applications/nrf_desktop/configuration/nrf54l15pdk_nrf54l15_cpuapp/child_image/mcuboot/prj.conf:28]: this commit (57805dbf95b3c66bcbd764f6bf32b055879ff989) can be dropped after we picked the right commit from upstream:
https://github.com/zephyrproject-rtos/zephyr/pull/71425/commits/0713655f830d04f5a647e72815b564a3135335ce
which has the node disabled.

**Review by tejlmand — APPROVED** [review]
Approved, but one commit can be dropped.

File: applications/nrf_desktop/configuration/nrf54l15pdk_nrf54l15_cpuapp/child_image/mcuboot/prj.conf (hunk 8 above)
Comment (gmarull) [reply applications/nrf_desktop/configuration/nrf54l15pdk_nrf54l15_cpuapp/child_image/mcuboot/prj.conf]: dropped

**Review by nordicjm — APPROVED** [review]

**Review by thst-nordic — APPROVED** [review]

**Pull request merged**